### ISSUE TITLE
UI Test cases failing only in my mac but not for others

### ISSUE THREAD

**Issue by BeingShaashi** [vuejs/vue-jest#303]
Hi,

This is a super weird thing, that I am facing, I have cloned a VueJS repo, it has no extra config, just npm install and go.
Before committing the code, we have added a prehook to check all UI Test cases are passing or not.

The code works well, in everyone else Mac or windows, all UI Cases are passing.
I got a new mac from my company and just installed node 14.15.1 via NVM. 

Then I got the code, the code is working fine, I am also able to create the build.
But when I am running a unit test, around 40% of the UI cases are Failing, and that is what is super weird.

I have already tried different ways, installing different versions of NodeJS, Clearing cache, installing exact versions as per package JSON, etc., etc. but still, tests are getting failed.

I am attaching a screenshot of a few errors, which only failing in my machine.

Please help me solve this error.

<img width="997" alt="Screenshot 2020-11-21 at 9 21 14 AM" src="https://user-images.githubusercontent.com/12844625/99866891-f64c6a00-2bda-11eb-924b-929753f5fa70.png">

**Comment by lmiller1990** [comment]
Does your project have a package.lock or yarn-lock or something? It sounds like you have different dependencies on your new machine, maybe? Is everything else the same (node version, etc)?

I am guessing you cannot share this repo?

**Comment by BeingShaashi** [comment]
Yes can't share the code repo.
It has package-lock.json file.

The point to note is - the application runs perfectly and as expected. Also
I am able to create the build with 'npm run build' command, so I have all
needed packages installed.

Problem comes only when testing UI components. It every cases were failing
the. it would have different case. But out of 375 only 117 cases are
failing rest are passed.

And this is only in my machine and for other all 375 cases are passing.

**Comment by lmiller1990** [comment]
Please share your full `mount` function?

**Comment by BeingShaashi** [comment]
Hi,

So after alot of debugging, I got the reason why the test cases were
failing only in my reason.

I was checking through each node_modules installed and found that the
"puppeteer" npm is missing from my node_modules which downloads chromium
headless.

So whenever I was doing npm install it was installing everything except the
puppeteer, hence my application was working fine but test cases were
failing.

I did "npm config rm https-proxy" and again npm install, then finally it
was able to install the puppeteer and chromium.
And then every test case passed as needed.

Though It cant be exact reason why npm install didnt installed puppeteer
npm, it is just based on my obervation, but in the end the workaround
worked.

Thanks
